packages/vscode-extension/src/devices/DeviceManager.ts

```diff
@@ -51,10 +51,14 @@ export class DeviceManager implements Disposable, DeviceManagerInterface {
   }
 
   public async acquireDevice(deviceInfo: DeviceInfo) {
-    if (deviceInfo.platform === Platform.IOS) {
+    if (deviceInfo.platform === DevicePlatform.IOS) {
+      if (process.platform !== "darwin") {
```

Comment:
Replace with helper. I'd actually throw here, no one should ask for iOS device on non-macOS.